### DISCUSSION TITLE
Bagaimanakah membuat heatmap dengan pengelompokan wilayah ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#629] [Q&A] [answered]
Bagaimanakah membuat heatmap dengan pengelompokan wilayah ? sedangkan ditiap wilayah tsb (3 wilayah) pada setiap jamnya terjadi transaksi yg berbeda-beda.

Jika dilakukan code dan hasilnya seperti berikut ;
<img width="456" alt="image" src="https://user-images.githubusercontent.com/123436963/231666089-879f0c66-2a06-401e-b09c-d22239442fa7.png">

dg hasil ;
<img width="674" alt="image" src="https://user-images.githubusercontent.com/123436963/231666344-4ef2ddc1-2d30-4f48-ae64-c7e6fb94ad67.png">

dari hasil tsb diatas, Bagaimanakah membuat heatmap dengan pengelompokan pada masing-masing wilayah (3 wilayah) tsb ?

Terimakasih

**Comment by ramdhinafinita** [comment, marked as answer]
Dear pak @Micoalgo 
Silahkan lakukan aggregasi data terlebih dahulu sebelum melakukan pembuatan heatmap yang diinginkan. 
Data hmp yang digunakan tidak hanya kolom area dan count, tetapi harap memasukkan kolom hournya berdasarkan kolom datetime nya. 

Namun, sebelum itu  sudah melakukan proses pembuatan kolom targetnya agar saat dibuat heatmap berdasarkan kolom aggregasi kolom targetnya. 
Mungkin bisa dilampirkan hasil read object sc_nodrive nya pak. Terimakasih

**Comment by Micoalgo** [comment]
Alhamdulillah di sesi mentoring siang ini juga sdh dibahas dan direview
hasilnya oleh Mba Fiqey. Terimakasih masukannya Mba Fini
